### ISSUE TITLE
Feature Request - iOS Safari Extension for iPhone/iPad

### ISSUE THREAD

**Issue by redleghicks** [yuanzhoucq/Tab-Space#98]
I love Tab Space!  Just asking for an iPhone/iPad extension - I would definitely pay again to have that feature!  Thanks!

**Comment by jpufferfish** [comment]
I agree. I have yet to find a “tab saving” extension on there that RESTORES saved tabs. Right now I’m resorting to “share tabs” by “Denk Alexandru”, but that seems to only export tabs to a url list, not import or restore. If you know of an app that does this (on mobile), feel free to let me know.

But…if your app syncs between separate macbooks via iCloud, I think you could skyrocket the convenience/uniqueness of your app by introducing a mobile version that syncs with the desktop version. I'd be willing to pay for a separate mobile version, similar to how the famous “Stopthemadness” extension separates mobile and desktop versions on the app store.

**Comment by wes1993** [comment]
There are some news here? alternative solutions?

**Comment by yuanzhoucq** [comment]
The iOS app is finally there with an extension included. Thanks for the suggestion!
